spirv_common.hpp

```diff
@@ -708,6 +708,7 @@ struct Meta
 	struct Decoration
 	{
 		std::string alias;
+        std::string qualified_alias;
```

Comment:
MSL disallows inputs and outputs (incl uniforms) to be defined as globals. All input/output vars must be passed as structure members. The qualified_alias iVar allows a type to be assigned an alternate name that includes the input/output structure it has been moved to.